setup.py

```diff
@@ -16,7 +16,7 @@ def load_requirements(use_case):
 
 setup(
     name="flexmeasures",
-    description="FlexMeasures - A free platform to build services for real-time optimization of flexible energy.",
+    description="FlexMeasures - a tool for building real-time energy flexibility services, rapidly and scalable.",
```

Comment:
If these 3 files constitute the repo's main copy, I suggest we keep the word "platform" in all three. So here I would write:

`description="The *FlexMeasures Platform* - a tool for building real-time energy flexibility services, rapidly and scalable."`

And in line 44:

```
The *FlexMeasures Platform* is a tool for building real-time energy flexibility services, rapidly and scalable.
```

In all other places (in code) besides these 3, we can simply write "FlexMeasures".